cmd/zar/stat/command.go

```diff
@@ -0,0 +1,78 @@
+package stat
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"os"
+
+	"github.com/brimsec/zq/archive"
+	"github.com/brimsec/zq/cmd/zar/root"
+	"github.com/brimsec/zq/emitter"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zio"
+	"github.com/mccanne/charm"
+)
+
+var Stat = &charm.Spec{
+	Name:  "stat",
+	Usage: "stat [options]",
+	Short: "archive component statistics",
+	Long: `
+"zar stat" generates a ZNG stream with information about the chunks in
+an archive.
+`,
+	New: New,
+}
+
+func init() {
+	root.Zar.Add(Stat)
+}
+
+type Command struct {
+	*root.Command
+	root   string
+	format string
+}
+
+func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
+	c := &Command{Command: parent.(*root.Command)}
+	f.StringVar(&c.root, "R", os.Getenv("ZAR_ROOT"), "root directory of zar archive to walk")
+	f.StringVar(&c.format, "f", "table", "output format")
```

Comment:
For UX consistency please could you unify this with the zq docstring for `-f`, which is currently 

> -f format for output data [zng,ndjson,table,text,types,zeek,zjson,tzng] (default "zng")

Reply:
fyi, all of those will work fine except `types`, which requires special handling, so I won't include it.